README.md

```
@@ -89,6 +89,28 @@ A MCP server configuration that tells MetaMCP how to start a MCP server.
 }
 ```
 
+#### 🔐 **Environment Variables & Secrets (STDIO MCP Servers)**
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)